code/modules/reagents/reagents/medicine.js

```diff
@@ -72,21 +74,21 @@ Object.assign(Inacusiate.prototype, {
 class Cryoxadone extends Medicine { // /datum/reagent/medicine/cryoxadone
 	mob_life(dt) {
 		if(this.holder.c.CarbonMob.bodytemperature >= 0 && this.holder.c.CarbonMob.bodytemperature <= 99) { // At extreme temperatures (upgraded cryo) the effect is greatly increased.
-			//TODO M.status_flags &= ~DISFIGURED
+			this.holder.c.LivingMob.status_flags &= ~combat_defines.DISFIGURED;
```

Comment:
Does bitwise stuff work the same in JS as it does in DM?

Reply:
ofcourse it does why would one language change the meaning of the operators 🤔

Reply:
because who the fuck knows with DM